marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid

```diff
@@ -0,0 +1,55 @@
+---
+converter: markdown
+metadata:
+  title: Setting Up a User's Instance Permissions
+  description: This guide will help you give Instance permissions to users of the Partner Portal.
+slug: get-started/partner-portal/setting-up-users-instance-permissions
+searchable: true
+---
+This guide will help you give Instance permissions to users of the Partner Portal. Permissions include:
+
+* Being allowed to create new Instances 
+* Having read, write, or write and deploy access to different Instances 
+
+## Requirements
+To be able to set up Partner permissions for users on the Partner Portal, you have to have access to our Partner Portal, and the users you would like to give permissions to.
+
+* [Accessing Our Partner Portal](/get-started/setup/accessing-partner-portal)
+* [Inviting a New User to the Partner Portal](/get-started/partner-portal/inviting-new-user-to-partner-portal)
+
+## Steps
+
+Setting up Instance permissions is a two-step process:
+
+1. Grant permission to create new Instances
+2. Grant permissions to access Instances
+
+### Step 1: Grant permission to create new Instances
+
+Log in to our [Partner Portal](https://portal.apps.near-me.com). In the [list of users](https://portal.apps.near-me.com/users), select the user by clicking on its email address.
+
+On the user's information page, click on `Manage permissions`, then select the `Instances` tab. If the user doesn't have permission to create new Instances, you can grant permission by clicking on `Grant` under `Grant permission to create new Instances`. 
+
+If the user has permission to create new Instances, you can revoke this permission, by clicking on `Revoke` under `Revoke permission to create new Instances`. 
+
+### Step 2: Grant permissions to access Instances
+
+To grant permissions to access Instances, select the Instances you would like to give the user access to by checking their checkboxes.
+
+Select access level (`Read`, `Write`, or `Write and deploy`):
+* `Read`: The user can see the Instance, but can't modify it. 
+* `Write`: The user can see and modify the Instance, but can't deploy. E.g. the user can change configuration, but can't make changes in the codebase and deploy them.  
+* `Write and deploy`: The user can see, modify, and deploy to the Instance. 
+
+Once you've selected the permissions, click on `Submit`.
+
+You can check the user's permissions anytime under `This user has access to following instances` on the `Instances` tab of the user's information page. 
```

Comment:
"information" page, hmm, maybe "user permissions" page? I'm not sure.

Reply:
This one is before you get to the "user permissions" page, so I wouldn't call it that. Maybe "the selected user's page"? Would that be better?